[2025-04-24T19:04:41.126+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_prices_data_downloader.save_to_excel manual__2025-04-24T19:04:28.738339+00:00 [queued]>
[2025-04-24T19:04:41.134+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_prices_data_downloader.save_to_excel manual__2025-04-24T19:04:28.738339+00:00 [queued]>
[2025-04-24T19:04:41.135+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-24T19:04:41.145+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): save_to_excel> on 2025-04-24 19:04:28.738339+00:00
[2025-04-24T19:04:41.153+0000] {standard_task_runner.py:60} INFO - Started process 1666 to run task
[2025-04-24T19:04:41.156+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'raw_prices_data_downloader', 'save_to_excel', 'manual__2025-04-24T19:04:28.738339+00:00', '--job-id', '164', '--raw', '--subdir', 'DAGS_FOLDER/raw_price_data.py', '--cfg-path', '/tmp/tmpxb3i8cqn']
[2025-04-24T19:04:41.158+0000] {standard_task_runner.py:88} INFO - Job 164: Subtask save_to_excel
[2025-04-24T19:04:41.199+0000] {task_command.py:423} INFO - Running <TaskInstance: raw_prices_data_downloader.save_to_excel manual__2025-04-24T19:04:28.738339+00:00 [running]> on host 8291240c5d79
[2025-04-24T19:04:41.283+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_prices_data_downloader' AIRFLOW_CTX_TASK_ID='save_to_excel' AIRFLOW_CTX_EXECUTION_DATE='2025-04-24T19:04:28.738339+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-24T19:04:28.738339+00:00'
[2025-04-24T19:04:41.540+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/pymongo/synchronous/mongo_client.py:2709 UserWarning: MongoClient opened before fork. May not be entirely fork-safe, proceed with caution. See PyMongo's documentation for details: https://dochub.mongodb.org/core/pymongo-fork-deadlock
[2025-04-24T19:04:41.559+0000] {logging_mixin.py:188} INFO - File tatasteel.xlsx stored in MongoDB!
[2025-04-24T19:04:41.559+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-24T19:04:41.568+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=raw_prices_data_downloader, task_id=save_to_excel, execution_date=20250424T190428, start_date=20250424T190441, end_date=20250424T190441
[2025-04-24T19:04:41.608+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-24T19:04:41.626+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
